_posts/2023-07-11-nvidia-is-a-new-sponsor.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "NVIDIA Is A New Sponsor Of The Scikit-Learn consortium at the Inria Foundation"
+date: July 11, 2023
+
+categories:
+  - Funding
+tags:
+  - Sponsor
+
+featured-image: NVIDIAxsklearn.jpg
+
+postauthors:
+  - name: NVIDIA
+    website: https://nvidia.com/
```

Comment:
Please change to "https://developer.nvidia.com/gpu-accelerated-libraries"

Reply:
I think as a general link to "NVIDIA the company as a whole" nvidia.com is a good URL to use. In addition https://developer.nvidia.com/gpu-accelerated-libraries doesn't refer to RAPIDS where the work is happening/I'm part of.